app.R

```diff
@@ -4,4 +4,5 @@
 
 # pkgload::load_all(export_all = FALSE,helpers = FALSE,attach_testthat = FALSE)
 options( "golem.app.prod" = TRUE)
+i18n <- shiny.i18n::Translator$new(translation_json_path = "data/translation.json")
```

Comment:
@efcaguab this line is in the wrong place, I have just put it there at the moment at your knowledge. The issue here is that when you run this line and save the object i18n into the environment before running the app it works. Otherwise, it doesn't find i18n and fails.